code/modules/mining/lavaland/megafauna_loot.dm

```diff
@@ -439,7 +439,10 @@
 		return
 	if(pixel_x != base_pixel_x || pixel_y != base_pixel_y)
 		animate(src, 0.2 SECONDS, pixel_x = base_pixel_y, pixel_y = base_pixel_y, flags = ANIMATION_PARALLEL)
-	Move(get_step_multiz(src, direction), direction)
+	if(direction == UP || direction == DOWN)
+		zMove(direction)
+	else
+		Move(get_step(src, direction))
```

Comment:
why not your new thing

Reply:
was tired when i made this and worried that changing this from move to step would break something, tested it now and turns out that step actually seems to work better so yeah I've changed this to use the new proc instead now